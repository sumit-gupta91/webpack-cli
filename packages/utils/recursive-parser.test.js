"use strict";

const defineTest = require("./defineTest");


defineTest(
	__dirname,
	"init",
	"fixture-1",
	"entry",
	{
		objects: "are",
		super: [
			"yeah",
			{
				test: new RegExp(/\.(js|vue)$/),
				loader: "'eslint-loader'",
				enforce: "'pre'",
				include: ["customObj", "'Stringy'"],
				options: {
					formatter: "'someOption'"
				}
			}
		],
		nice: "':)'",
		man: "() => duper"
	}
);
