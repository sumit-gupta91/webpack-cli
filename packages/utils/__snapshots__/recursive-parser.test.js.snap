// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init transforms correctly using "fixture-1" data 1`] = `
"module.exports = {
  entry: {
    objects: are,

    super: [yeah, {
      test: /\\\\.(js|vue)$/,
      loader: 'eslint-loader',
      enforce: 'pre',
      include: [customObj, 'Stringy'],

      options: {
        formatter: 'someOption'
      }
    }],

    nice: ':)',
    man: () => duper
  }
};"
`;
